### PR TITLE
Further readme edits

### PR DESCRIPTION
**Pull request by exbinary**
See commit messages below for a list of the high level changes.
Probably also easier to review the commits separately.
Though, TBH, an even better way is probably to click around the branch tree itself:
https://github.com/rubyforgood/mutual-aid/tree/further-readme-edits.

### CODE REVIEW

File: SEEDING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (maebeale) [inline SEEDING.md:1]: May we add a header here? And a link back to README, like in the others?

File: README.md [hunk 2]
```diff
@@ -68,7 +68,10 @@ Visit our [DEPLOYMENT.md](DEPLOYMENT.md) guide for more information on deploying
 * [Code of conduct](https://github.com/rubyforgood/code-of-conduct)
```
Comment (maebeale) [inline README.md:68]: <3<3<3

File: README.md [hunk 3]
```diff
@@ -60,6 +60,16 @@ Please, get involved! All of our communities could benefit from more resiliency
 We wrote up a [Glossary of terms](http://mutual-aid-demo.herokuapp.com/admin/glossary)  that's available in the demo instance once you're logged in.
 You can also review it [in code](blob/main/app/views/admin/glossary.html.erb).
 
+## Deploying the app
+Visit our [DEPLOYMENT.md](DEPLOYMENT.md) guide for more information on deploying the app.
+
+## More guides and documentation
```
Comment (maebeale) [inline README.md:66]: LOVE THIS SO MUCH!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -36,3 +50,6 @@ Some things that will increase the chance that your pull request is accepted:
 * Use Rails idioms and helpers
 * Include tests that fail without your code, and pass with it
 * Update the documentation, the surrounding one, examples elsewhere, guides, whatever is affected by your contribution
+
+## Lastly...
+Bask in the glory that is you! ✨🙏🏾
```
Comment (maebeale) [inline CONTRIBUTING.md:55]: YESSSSS!!!!!

File: README.md [hunk 5]
```diff
@@ -50,6 +50,16 @@ In addition to the background links in the 'What is mutual aid?' section, here's
 
 Please, get involved! All of our communities could benefit from more resiliency and connection.
 
+## Check out the demo
```
Comment (maebeale) [inline README.md:53]: <3 love this

**Review by maebeale — APPROVED** [review]
LOVE>ALL>OF>THIS

**Comment by gingeralbertson** [comment]
The changes are great!! Looks and flows perfectly :)

When I was testing links I found the below. I wasn't sure about if or how
to fix.

   - in "Contribute" the "this guide" link isn't working
   - error message for "Glossary of Terms" and "in code"
   - error message for "DEPLOYMENT.md" link
   - did we want Customization (WIP) under "More guides and documentation"
   to link to something?
   - error message for "browse some of our earliest notes" under "Who we
   are"




Ginger Albertson
www.linkedin.com/in/gingeralbertson
+1 917 553 9303

**Comment by exbinary** [comment]
> When I was testing links I found the below.
Thank you for unearthing these! Fixing.

> did we want Customization (WIP) under "More guides and documentation" to link to something?
I think this is a placeholder for future documentation.

**Pull request merged**